iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerLexicalConstant.java

```diff
@@ -52,7 +52,7 @@ public class DockerLexicalConstant {
    */
   private static final String PERMITTED_EMBEDDED_DOLAR = "(?:\\$(?!\\{|" + VAR_IDENTIFIER_START + "))";
 
-  private static final String NON_SPECIAL_CHARACTERS = "(?:[^\"\\\\$\\{])";
+  private static final String NON_SPECIAL_CHARACTERS = "(?:[^\"\\\\$])";
```

Comment:
I adapted this reggex as it was preventing from parsing such thing : `"{command}"`